comparisontool/tests/management/commands/test_load_school.py

```diff
@@ -0,0 +1,29 @@
+import tempfile
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from comparisontool.models import School
+
+
+class LoadSchoolTests(TestCase):
+    def test_load_csv(self):
```

Comment:
Similar thing as above.  I see the pattern here, but I think if someone's looking at a single test it'd help them understand what it's doing right away if the name of the test is more descriptive.